src/olympia/zadmin/forms.py

```diff
@@ -239,64 +239,3 @@ class CompatForm(BaseCompatForm):
     _ratio_choices = [('%.1f' % (x / 10.0), '%.0f%%' % (x * 10))
                       for x in xrange(9, -1, -1)]
     ratio = forms.ChoiceField(choices=_ratio_choices, required=False)
-
-
-class GenerateErrorForm(happyforms.Form):
-    error = forms.ChoiceField(choices=(
-        ['zerodivisionerror', 'Zero Division Error (will email)'],
-        ['iorequesterror', 'IORequest Error (no email)'],
-        ['heka_statsd', 'Heka statsd message'],
-        ['heka_json', 'Heka JSON message'],
-        ['heka_cef', 'Heka CEF message'],
-        ['heka_sentry', 'Heka Sentry message'],
-        ['amo_cef', 'AMO CEF message']))
```

Comment:
Yeah I think was mostly around when heka and CEF was coming in, we needed it to check the error went where it was meant to go to. Remove it and lets see how things go.